sky/engine/public/platform/sky_settings.h

```diff
@@ -17,6 +17,7 @@ struct SkySettings {
   bool start_paused = false;
   bool enable_dart_checked_mode = false;
   bool trace_startup = false;
+  bool complete_timeline = false;
```

Comment:
What is "complete timeline" ?  We should have a better name than this.  Why would I ever want an incomplete timeline?